lib/bitcoin/BitcoinClient.ts

```diff
@@ -28,7 +29,8 @@ export default class BitcoinClient {
     bitcoinRpcPassword: string | undefined,
     bitcoinWalletImportString: string,
     private requestTimeout: number,
-    private requestMaxRetries: number) {
+    private requestMaxRetries: number,
+    private transactionFeeMarkupPercentage: number) {

```

Comment:
One concern that I have is that this percentage is only applied to the 'sidetree' transactions. Any other transactions such as lock/relock etc don't honor this percentage. So we need to decide between:
1. Either every transaction has this markup.
1. Or rename this variable to denote that this markup is ONLY for the sidetree transactions.

Reply:
Will rename to only apply to sidetree transactions as discusses yesterday.